azure/durable_functions/models/OrchestratorState.py

```diff
@@ -76,6 +85,8 @@ def to_json(self) -> Dict[str, Union[str, int]]:
         """
         json_dict: Dict[str, Union[str, int]] = {}
         add_attrib(json_dict, self, '_is_done', 'isDone')
+        if self._replay_schema != ReplaySchema.V1:
```

Comment:
Is there a reason we don't want this for V1?

Reply:
I can't remember the exact reason on the top of my head but, thinking about it now, I believe it broke various tests due to the unexpected new field. I believe a big refactor of the testing utilities is overdue before I can add this field without breaking most tests

Reply:
But now having it there shouldn't break anything, due to how the C# changes were implemented